consul/consul_test.go

```diff
@@ -585,10 +576,18 @@ func TestMarathonTaskToConsulServiceMapping(t *testing.T) {
 	assert.Equal(t, "127.0.0.6", service.Address)
 	assert.Equal(t, []string{"marathon", "public"}, service.Tags)
 	assert.Equal(t, 8090, service.Port)
-	assert.NotNil(t, "http://127.0.0.6:8090/api/health", service.Check)
-	assert.Empty(t, service.Checks)
-	assert.Equal(t, "http://127.0.0.6:8090/api/health", service.Check.HTTP)
-	assert.Equal(t, "60s", service.Check.Interval)
+	assert.Nil(t, service.Check)
+	assert.Equal(t, 4, len(service.Checks))
+
+	assert.Equal(t, "http://127.0.0.6:8090/api/health?with=query", service.Checks[0].HTTP)
+	assert.Equal(t, "https://127.0.0.6:8090/secure/health?with=query", service.Checks[1].HTTP)
+	assert.Equal(t, "127.0.0.6:8443", service.Checks[2].TCP)
+	assert.Equal(t, "echo 1", service.Checks[3].Script)
+
+	assert.Equal(t, "60s", service.Checks[0].Interval)
+	assert.Equal(t, "50s", service.Checks[1].Interval)
+	assert.Equal(t, "40s", service.Checks[2].Interval)
+	assert.Equal(t, "30s", service.Checks[3].Interval)
```

Comment:
How about comparing whole slices instead of specific fileds? I mean one assert for all checks?